identity-service/src/models/authMigrations.js

```diff
@@ -1,20 +0,0 @@
-'use strict'
```

Comment:
@SidSethi can we write a migration to drop from the db

Reply:
done